released/packages/coq-monae/coq-monae.0.9.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/affeldt-aist/monae"
+dev-repo: "git+https://github.com/affeldt-aist/monae.git"
+bug-reports: "https://github.com/affeldt-aist/monae/issues"
+license: "LGPL-2.1-or-later"
+
+synopsis: "Monads and equational reasoning in Coq"
+description: """
+This Coq library contains a hierarchy of monads with their laws used
+in several examples of monadic equational reasoning."""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install_full"]
+depends: [
+  "coq" { (>= "8.19" & < "8.21~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "2.3.0") }
+  "coq-mathcomp-fingroup" { (>= "2.3.0") }
+  "coq-mathcomp-algebra" { (>= "2.3.0") }
+  "coq-mathcomp-solvable" { (>= "2.3.0") }
+  "coq-mathcomp-field" { (>= "2.3.0") }
+  "coq-mathcomp-analysis" { (>= "1.9.0") }
+  "coq-infotheo" { >= "0.9.1"}
+  "coq-paramcoq" { >= "1.1.3" & < "1.2~" }
+  "coq-hierarchy-builder" { >= "1.7.0" }
+  "coq-equations" { >= "1.3" & < "1.4~" }
+]
+
+tags: [
+  "keyword:monae"
+  "keyword:effects"
+  "keyword:probability"
+  "keyword:nondeterminism"
+  "logpath:monae"
```

Comment:
Looking at the repository, I see that the `_CoqProject` file contains
```
theories/core/preamble.v
theories/core/hierarchy.v
...
-R . monae
```
which means that all the files are actually installed under the logical path `monae.theories`. Is that right?

Also, I see that there is a second logical path in a separate directory.

```suggestion
  "logpath:monae.theories"
  "logpath:monaeImpredicativeSet"
```